lib_eio_windows/low_level.ml

```diff
@@ -138,6 +138,12 @@ let read_link ?dirfd path =
   in_worker_thread @@ fun () ->
   Eio_unix.Private.read_link dirfd path
 
+let chown ?dirfd ~follow:_ ~uid ~gid path =
+  in_worker_thread @@ fun () ->
+  match dirfd with
+  | None -> failwith "Chown is unsupported on Windows"
```

Comment:
Could just use `Unix.chown` here.

Reply:
Hmm I'm wondering whether `chown` should be in the cross-platform API at all. 

`Unix.chown` isn't implemented on Windows: 
https://github.com/ocaml/ocaml/blob/c484c6932fa2eae03ba0f5a7dbdb26e3eee65eb0/otherlibs/unix/unix_win32.ml#L450 -- it's no implemented in python's `os.chown` either?

Do we have a rule for what makes it into the cross-platform API and what doesn't ? I mean we already have `File.Unix_perm.t`.

Reply:
I think it's OK to keep it. We might want a proper operation-not-supported error at some point.